test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -267,7 +280,15 @@ public void initClient() throws IOException {
                     );
                 }
             }
+
+            testFeatureService = new TestFeatureService(
+                List.of(new RestTestLegacyFeatures()), // TODO: add new ESRestTestCaseHistoricalFeatures() too
```

Comment:
Are there ticket references for these todos?

Reply:
Yes, added to the TODO